src/Tools/ExternalAccess/FSharp/Internal/Editor/FSharpNavigationBarItemService.cs

```diff
@@ -68,7 +68,7 @@ private static NavigationBarItem ConvertToNavigationBarItem(FSharpNavigationBarI
                     item.Text,
                     FSharpGlyphHelpers.ConvertTo(item.Glyph),
                     item.Spans,
-                    item.ChildItems.Select(x => ConvertToNavigationBarItem(x)).ToList(),
+                    item.ChildItems?.Select(x => ConvertToNavigationBarItem(x)).ToList(),
```

Comment:
This _might_ throw, it may be preferred to return null from this function.

Reply:
This is just a conversion and private. `item` really should never be null because the mapping happens on a collection.